[ISSUE #21] partial_ratio incorrectly using substr causing ratio to be incorrect

NickDunning opened the pull request:
Fix for https://github.com/tmplt/fuzzywuzzy/issues/21

NickDunning committed:
Update fuzzywuzzy.cpp

corrected partial_ratio function to fix how were using substr which was causing ratio to be incorrect.

tmplt commented:
Thanks. Unfortunately I do not have time to verify this. Can you amend the commit message to something more descriptive, like "partial_ratio: fix substr use", and I'll merge?

NickDunning committed:
partial_ratio: fix substr use

corrected partial_ratio function to fix how were using substr which was causing ratio to be incorrect.

NickDunning commented:
> Thanks. Unfortunately I do not have time to verify this. Can you amend the commit message to something more descriptive, like "partial_ratio: fix substr use", and I'll merge?

@tmplt done
